src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/CreateArticleModalComponentObject.java

```diff
@@ -85,8 +86,14 @@ private void chooseLayout(String layout) {
     }
   }
 
-  public void verifyCreateNewArticleModal() {
-    wait.forElementVisible(addAPageButton);
+  public boolean isCreateNewArticleModalVisible() {
+    try {
+      wait.forElementVisible(addAPageButton);
+      return true;
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)